float64[16] resulting_transformation
float64 peak_height
float64


